platform/web/os_web.cpp

```diff
@@ -174,7 +174,10 @@ void OS_Web::add_frame_delay(bool p_can_draw) {
 #endif
 }
 
-void OS_Web::vibrate_handheld(int p_duration_ms) {
+void OS_Web::vibrate_handheld(int p_duration_ms, float p_amplitude) {
+	if (p_amplitude != -1.0) {
+		WARN_PRINT_ONCE("Cannot change amplitude on web build!");
+	}
```

Comment:
I would not warn here, as I think it's going to be a bother if trying to make a cross-platform project that uses amplitude on supported platforms and would be fine with just have the default one used on web.